### ISSUE TITLE
Search: English text appears in the French result page

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#123]
URL: https://hcdev.openplus.ca/fr/avis-rappel/avis-rappel-aliments-allergene-rappel-certains-produits-vegetaliens-et-vegetariens
User Agent: Chrome
Language: French

Issue Description: There is English text appears in the French searching result page.
![image](https://user-images.githubusercontent.com/88776404/137168910-7b5fd77f-1572-4613-b9bf-27c547c21a98.png)

Steps to reproduce:

1. Navigate to https://hcdev.openplus.ca/en/search/site
2. Enter "Veggie" to the Search field and activate the "Search" button
3. Switch to French page
3. Select the second item
4. English text appears in the French page

Expected results: On the French page, all text should be translated into French.

**Comment by davidgotlieb** [comment]
@XiuJuanXia These are brand names of products, so this is a data entry question not a system question. In this case Brands are usually unilingual, so you will have English brands on the French side.

**Comment by XiuJuanXia** [comment]
@davidgotlieb Is it possible to add lang="en" for the English brand names of products that appear in the French side?

**Comment by davidgotlieb** [comment]
@XiuJuanXia Definitely not in the title, we looked into that. No HTML code is allowed, it can break things elsewhere in Drupal and looks bad.

For the Brand, it's just a plain text field as well so if you include HTML it displays that code

**Comment by XiuJuanXia** [comment]
@davidgotlieb Thanks for explanation. I also noticed that the alt attribute for the graphics are in English on the French page. Would it possible to add lang="en" for the graphics so that screen readers can read in English?

**Comment by mmiron-HC** [comment]
@davidgotlieb This system, as I understand it, has a very high profile with high visibility. We need to ensure that everything is as accessible as possible. This is an open source platform, why do they not provide a solution rather than shutting it down?
This may seem like a banal issue, however it comes with a serious severity. This impacts all users and will cause confusion with people who have cognitive difficulties and folks who use assistive tech like screen readers. There must be a solution.

**Comment by davidgotlieb** [comment]
@mmiron-HC I'm sure they can change it, but we're at a point where we are prioritizing work. The simple solution, which would be to simply allow HTML in those fields, is a known problem that breaks systems. As a result, that won't work.

I suspect they would have to do some pretty major development to basically fix a known issue in Drupal. So yes, that can be done and would contribute to an improved open source tool, but it would also have impacts on the work schedule, costs and/or what features we can deliver later on. So we need to prioritize their development time.

If you tell us that having HTML in titles and other fields is critical so that the lang attribute can be used, and we can't go live, than that affects the priority. But until now we had no indication of the significance/impact of this issue, so I was simply explaining why  English is showing up on the French, and that it isn't an error but rather than the content is actually supposed to be English.

**Comment by davidgotlieb** [comment]
> @davidgotlieb Thanks for explanation. I also noticed that the alt attribute for the graphics are in English on the French page. Would it possible to add lang="en" for the graphics so that screen readers can read in English?

@XiuJuanXia I'm not sure I know what you mean about the alt attributes on the graphics? Could you perhaps do a separate item with screen caps so we can send it to OpenPlus?

**Comment by XiuJuanXia** [comment]
@davidgotlieb A separate issue has been created https://github.com/mmiron-HC/WST-RSAMS/issues/124.

**Comment by ndolter** [comment]
Closing the ticket as it is no longer an issue.  Unable to see the sub-title as seen in the 1st screenshot above.
